guardian/test/utils/junit_xml/element_parser/test_case_error_parser_test.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:guardian/utils/junit_xml/junit_xml.dart';
+import 'package:test/test.dart';
+import 'package:xml/xml.dart';
+
+import '../test_utils/xml_string_builder_util.dart';
+import '../test_utils/xml_string_parse_util.dart';
+
+void main() {
+  group('TestCaseErrorParser', () {
+    const text = 'Random test case error';
```

Comment:
Nit: 
```suggestion
    const text = 'Test case error';
```
to not confuse reader with "Random"